scripts/install-cni.sh

```diff
@@ -34,11 +34,11 @@ if [ -w /host/etc/cni/net.d ]; then
   cni_spec=$(echo ${CNI_SPEC_TEMPLATE:-} | sed -e "s#@ipv4Subnet#[{\"subnet\": ${ipv4_subnet:-}}]#g")
 
   if [ "$ENABLE_PRIVATE_IPV6_ACCESS" = true ]; then
-    node_ipv6_addr=$(curl -s -k --fail "http://metadata.google.internal/computeMetadata/v1/instance/network-interfaces/0/?recursive=true" -H "Metadata-Flavor: Google" | jq '.ipv6s[0]') ||:
+    node_ipv6_addr=$(curl -s -k --fail "http://metadata.google.internal/computeMetadata/v1/instance/network-interfaces/0/?recursive=true" -H "Metadata-Flavor: Google" | jq -r '.ipv6s[0]' ) ||:
```

Comment:
what does jq do ?

Reply:
jq is a json parser than Jing told me about :)
This change adds the '-r' (aka --raw) param which removes the quotes from the returned string.

Reply:
interesting, does our container contain this tool? I didn't see it in the build script.

Reply:
It's added in the dockerfile here: https://github.com/GoogleCloudPlatform/netd/blob/master/Dockerfile.in#L19